validation/src/main/java/io/micronaut/validation/exceptions/ConstraintExceptionHandler.java

```diff
@@ -46,14 +47,20 @@
 @Requires(classes = {ConstraintViolationException.class, ExceptionHandler.class})
 public class ConstraintExceptionHandler implements ExceptionHandler<ConstraintViolationException, HttpResponse<JsonError>> {
 
+    private final JacksonConfiguration jacksonConfiguration;
+
+    public ConstraintExceptionHandler(JacksonConfiguration jacksonConfiguration) {
+        this.jacksonConfiguration = jacksonConfiguration;
+    }
```

Comment:
To avoid a breaking API change please restore the no argument constructor and annotate this one with `@Inject`